util/cron/test-slurm-gasnet-ibv.numa.bash

```diff
@@ -1,14 +1,13 @@
 #!/usr/bin/env bash
 #
 # Multi-node, multi-locale testing on a linux64 cluster with slurm-gasnetrun_ibv launcher:
-# test numa locale model with comm=gasnet on multilocale and examples.
+# test numa locale model with comm=gasnet against "examples"
 
 CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
 source $CWD/common-slurm-gasnet-ibv.bash
-source $CWD/common-numa.bash
 
-export CHPL_NIGHTLY_TEST_CONFIG_NAME="slurm-gasnet-ibv.numa"
+export CHPL_LOCALE_MODEL=numa
 
-export GASNET_QUIET=Y
```

Comment:
Thanks for checking...  Some other gasnet test configs have GASNET_QUIET. Some don't. As far as I could tell, it did nothing on chapcs.  Also, I had removed GASNET_QUIET from test-slurm-gasnet-ibv.fast.bash a few days ago #4740 and nothing bad happened.